src/main/java/com/example/CustomerService/advice/ExceptionHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.example.CustomerService.advice;
+
+import com.example.CustomerService.exception.CustomerNotFoundException;
+import com.example.CustomerService.exception.PasswordNotFoundEcxeption;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@RestControllerAdvice
+public class ExceptionHandler {
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @org.springframework.web.bind.annotation.ExceptionHandler(MethodArgumentNotValidException.class)
+    public Map<String, String> handleInvalidParameters(org.springframework.web.bind.MethodArgumentNotValidException ex){
+        Map<String, String> errorMap=new HashMap<>();
```

Comment:
Please create a generic response DTO like the request DTO we created. Map code gets redundant